docs/deprecations.md

```diff
@@ -0,0 +1,75 @@
+# Deprecations
+
+Sometimes a feature of Argo Workflows is deprecated.
+You should stop using a deprecated feature as it may be removed in a future minor or major release of Argo Workflows.
+
+To determine if you are using a deprecated feature the [`deprecated_feature`](metrics.md#deprecated_feature) metric can help.
+This metric will go up for each use of a deprecated feature by the workflow controller.
+This means it may go up once or many times for a single event.
+If the number is going up the feature is still in use by your system.
+If the metric is not present or no longer increasing are no longer using the monitored deprecated features.
+
+## `cronworkflow schedule`
+
+The spec field `schedule` which takes a single value is replaced by `schedules` which takes a list.
+To update this replace the `schedule` with `schedules` as in the following example
+
+```yaml
+spec:
+  schedule: "30 1 * * *"
+```
+
+is replaced with
+
+```yaml
+spec:
+  schedules:
+    - "30 1 * * *"
```

Comment:
These examples would likely be easier to understand quickly with a ```diff, i.e.:

```diff
-   schedule: "30 1 * * *"
+   schedules:
+     - "30 1 * * *"
```